policies/aws_account_policies/aws_resource_required_tags.py

```diff
@@ -1,19 +1,17 @@
 # REQUIRED_TAGS_MAPPINGS maps resource types to a set of tag keys required for that resource.
-# Example: REQUIRED_TAGS_MAPPINGS = {'AWS.EC2.Instance.Snapshot': {'TagOne', 'TagTwo'}}
+# Example: REQUIRED_TAGS_MAPPINGS = {'AWS.EC2.Instance.Snapshot': {'Owner', 'CreatedBy'}}
 # The above example would check all EC2 instances for the presence of tags keyed TagOne and TagTwo
-REQUIRED_TAGS_MAPPINGS = {}
+REQUIRED_TAGS_MAPPINGS = {
```

Comment:
can you set the default here to be the empty list, and mock  it out in the test settings? 

example [test-case yaml bit ](https://github.com/panther-labs/panther-analysis/blob/master/rules/github_rules/github_action_failed.yml#L59-L63)
example [in  rule()  handling for the mock](https://github.com/panther-labs/panther-analysis/blob/master/rules/github_rules/github_action_failed.py#L12-L14)